checkQC/config.py

```diff
@@ -91,12 +92,15 @@ def _get_matching_handler(self, instrument_and_reagent_type, read_length):
                 split_read_length = config_read_length.split("-")
                 low_break = int(split_read_length[0])
                 high_break = int(split_read_length[1])
-                if low_break < int(read_length) <= high_break:
+                if low_break <= int(read_length) <= high_break:
                     return self._config[instrument_and_reagent_type][config_read_length]["handlers"]
             else:
                 if int(read_length) == int(config_read_length):
                     return self._config[instrument_and_reagent_type][int(config_read_length)]["handlers"]
-        raise KeyError
```

Comment:
I think that the logging that we were previously doing in the `get_handler_config` method should go here, so that this error is communicated to the user. Right now I don't think any indication of this will show up to the user.